website/client/src/components/static/features.vue

```diff
@@ -137,6 +137,12 @@ export default {
   directives: {
     markdown: markdownDirective,
   },
+  mounted () {
+    this.$store.dispatch('common:setTitle', {
+      section: this.$t('help'),
```

Comment:
sorry, I don't think that for this static pages (except for FAQ). `help` is the right section, I'd just keep the page name, so in this case it would be Features | Habitica